### DISCUSSION TITLE
Howto add extra networks to mynetworks

### DISCUSSION THREAD

**Discussion by doets001** [docker-mailserver/docker-mailserver#2483] [Q&A] [answered]
I use PERMIT_DOCKER=container and so mynetworks is filled dynamically with the container IP. How can I add an extra network to the mynetworks and also keep the  PERMIT_DOCKER=container working.
I can overrule mynetworks by adding it to the postfix-main.cf, but then it will completely overwrite the value with what I add in the postfix-main.cf.

**Comment by casperklein** [comment, marked as answer]
Put something like this in your [user-patches.sh](https://docker-mailserver.github.io/docker-mailserver/edge/faq/#how-to-adjust-settings-with-the-user-patchessh-script) script should work:

```bash
CUSTOM_NETWORK="x.x.x.x/xx"
postconf -e "$(postconf | grep '^mynetworks =') $CUSTOM_NETWORK"
echo "$CUSTOM_NETWORK" >> /etc/opendmarc/ignore.hosts
echo "$CUSTOM_NETWORK" >> /etc/opendkim/TrustedHosts
```
